sagemaker-pipeline-parameterization/notebook.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        except Exception as e:
</pre><p>What would cause an exception in deleting the models and pipelines? How can customers manually delete these resources if the code throws exceptions?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3021/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>If there is no model package or SM Pipeline with the specified names, it throws an error.</p>

Reply:
<p>If the customer manually changes the model registry names, or the pipeline name when triggering an execution, and not in the variable that's set up top, this will ensure there is no error and that deleteable model packages/pipelines still gets cleaned up.</p>

Reply:
Okay, so this will handle edge cases and delete the resources as long as they still exist with expected names. Looks good to me.